docroot/profiles/custom/cgov_site/modules/custom/cgov_image/config/install/core.entity_view_mode.media.ncids_list_media.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
What is this template for? I see `ncids_collections_media` in use all over the place, but I don't understand what this is for.